addon/pods/components/rui-error-template/template.hbs

```diff
@@ -14,5 +14,14 @@
     <div class="rui-error-template-tagline">
       {{tagline}}
     </div>
+    {{#if link}}
+      <div class='rui-error-button'>
+        <a href={{link}}>
+          {{#rui-button size='lg'}}
```

Comment:
This emits a button tag so probably don't want to wrap it in an anchor tag.

I think instead of passing the link in, you should just create another conditional on this component that checks if the status is a 401 and just shows the login button.  That way, only the 401 errors will have the template and it never has to be passed in.